test/spec/provider/cloud-element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -640,6 +640,69 @@ describe('cloud-element-templates - ChangeElementTemplateHandler', function() {
 
     });
 
+
+    describe('update task type', function() {
+
+      const newTemplate = require('./task-template-elementType-1.json');
+
+      beforeEach(bootstrap(require('./task.bpmn').default, [ newTemplate ]));
+
+      it('execute', inject(function(elementRegistry) {
+
+        // given
+        const task = elementRegistry.get('Task_1');
+
+        // assume
+        expect(is(task, 'bpmn:ServiceTask')).to.be.true;
+
+        // when
+        changeTemplate(task, newTemplate);
```

Comment:
Proposal: Move this whole thing to our new public API (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/624).

It also returns the new task, out of the box.